src/main/java/com/internet/shop/model/User.java

```diff
@@ -2,9 +2,11 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import lombok.ToString;
 
 @Getter
 @Setter
+@ToString
```

Comment:
btw, you can read about 'Data' annotation

Reply:
okey, thx